invenio_app_rdm/config.py

```diff
@@ -665,7 +665,7 @@ def files_rest_permission_factory(obj, action):
 # ===============
 # See https://invenio-indexer.readthedocs.io/en/latest/configuration.html
 
-INDEXER_DEFAULT_INDEX = "rdmrecords-records-record-v6.0.0"
+INDEXER_DEFAULT_INDEX = "rdmrecords-records-record-v7.0.0"
```

Comment:
will the index be created automatically after deployment or we need to add this to an upgrade recipe>

Reply:
We will create the index and do reindexing before but Alex and I discussed that records and communities indices bump up need to be strategised to prevent downtime if possible.